packages/molenc/molenc.15.4.0/opam

```diff
@@ -0,0 +1,75 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/molenc"
+bug-reports: "https://github.com/UnixJunkie/molenc/issues"
+dev-repo: "git+https://github.com/UnixJunkie/molenc.git"
+license: "BSD-3-Clause"
+build: ["dune" "build" "-p" name "-j" jobs]
+install: [
+  # ["cp" "bin/molenc_ph4_type_atoms.py" "%{bin}%/molenc_ph4_type_atoms.py"]
+  ["cp" "bin/molenc.sh" "%{bin}%/molenc.sh"]
+  ["cp" "bin/molenc_common.py" "%{bin}%/molenc_common.py"]
+  ["cp" "bin/molenc_lean.py" "%{bin}%/molenc_lean.py"]
+  ["cp" "bin/molenc_lizard.py" "%{bin}%/molenc_lizard.py"]
+  ["cp" "bin/molenc_scan.py" "%{bin}%/molenc_scan.py"]
+  ["cp" "bin/molenc_scan.sh" "%{bin}%/molenc_scan.sh"]
+  ["cp" "bin/molenc_standardize.py" "%{bin}%/molenc_standardize.py"]
+  ["cp" "bin/molenc_type_atoms.py" "%{bin}%/molenc_type_atoms.py"]
+  ["cp" "bin/smi2png.py" "%{bin}%/molenc_smi2png.py"]
+  ["cp" "bin/molenc_smisur.py" "%{bin}%/molenc_smisur.py"]
+  ["cp" "bin/molenc_panascan.py" "%{bin}%/molenc_panascan.py"]
+]
+depends: [
+  "batteries"
+  "bst"
+  "conf-graphviz"
+  "conf-python-3"
+  "conf-rdkit"
+  "cpm"
+  "dokeysto_camltc"
+  "dolog" {>= "5.0.0"}
+  "dune" {>= "1.11"}
+  "minicli" {>= "5.0.0"}
+  "ocaml"
```

Comment:
```suggestion
  "ocaml" {>= "4.07"}
```
Module `Float` was introduced in 4.07. Do you want to support older compilers or this is fine?